gtk/gtk.tree-model.lisp

```diff
@@ -836,9 +835,8 @@
   @short{Prepends a new @arg{index} to a @arg{path}.}
 
   As a result, the depth of the @arg{path} is increased."
-  (let ((path (gtk-tree-path-copy path)))
-    (%gtk-tree-path-prepend-index path index)
-    path))
+ (%gtk-tree-path-prepend-index path index)
```

Comment:
Would be good if you could also update the `@return` annotation then, since it will return the exact same path now.